scripts/build-archive/build-archive.go

```diff
@@ -22,28 +22,30 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var pluginFiles = []string{
+	"dist/",
+	"TYPE",
```

Comment:
I'm wondering if it makes sense since these metadata info can be found in the `package.json` already.
Perhaps we can include the `package.json` in the archive as well.

Reply:
Having a specific json file for the plugin could be interesting if we have info that cannot be found in the `package.json` or in the file `mf-manifest.json`

Reply:
Then we could use the package.json with a custom field `perses` if we need to include more metadata as the package.json should hold already the version of the plugin and author. We can add there the plugin type like:

```
perses:{
  pluginType:"panel"
}
```

WDYT?

Reply:
ah yeah I like this idea. Will make the changes ! Thanks 🙏

Reply:
done